### PR TITLE
LG-11734 Add faqs

### PR DESCRIPTION
**Pull request by Sgtpluck**
Ticket:
https://cm-jira.usa.gov/browse/LG-11734

### CODE REVIEW

**Review by Jeremy1026 — APPROVED** [review]

**Review by nprimak — APPROVED** [review]
Some of these feel a little redundant with what we have on the [help center pages](https://www.login.gov/help/get-started/overview/) but it probably doesn't hurt to reiterate things. LGTM!

File: _pages/user-experience/faq-content.md [hunk 1]
```diff
@@ -15,9 +15,60 @@ sidenav:
...
-  - text: FAQ content 
+  - text: FAQ content
     href: "user-experience/faq-content/"
 
 ---
 
-## FAQ Content
+## Add FAQ content to inform users about Login.gov (Optional)
+
+When you first integrate with Login.gov, we recommend adding frequently asked questions (FAQ) content to ensure that users are able to understand that your application uses Login.gov as a sign-in service.. The next few sections outline plain language for agencies to leverage when creating that content.
```
Comment (karenstrother) [inline _pages/user-experience/faq-content.md:25]: extra period after the first sentence

File: _pages/user-experience/faq-content.md [hunk 2]
```diff
@@ -15,9 +15,60 @@ sidenav:
...
+
+Login.gov requires you to use a multifactor authentication method. You can select to receive a code via SMS/Text or phone call or use an authentication application, government PIV/CAC card, face or touch unlock, security key or download backup codes. We encourage you to add more than one authentication method to your account to avoid getting locked out if you ever lose an authentication method. <a
+    class="usa-link usa-link--external"
+    rel="noreferrer"
+    target="_blank"
+    href="https://www.login.gov/help/get-started/authentication-options/"
+    >Learn more about the different authentication methods you can use</a>.
+
+### How do I sign in to [APPLICATION] after I create my Login.gov account?
+Every time you click on Sign In on the APPLICATION site, you’ll go to the Login.gov sign in page. Enter your Login.gov email address, password and use your two-factor authentication method. . Once you enter this information correctly, you’ll be signed in and automatically directed back to the APPLICATION site.
```
Comment (karenstrother) [inline _pages/user-experience/faq-content.md:46]: Extra period after the second sentence.

File: _pages/user-experience/faq-content.md [hunk 3]
```diff
@@ -15,9 +15,60 @@ sidenav:
...
+
+### Can I use my existing [APPLICATION] username and password to sign in?
+No. Your old username and password will no longer work. You must create a new account or use your existing Login.gov to sign in. We’ll connect your new Login.gov account with your existing [APPLICATION] profile during the process.
```
Comment (karenstrother) [inline _pages/user-experience/faq-content.md:49]: Should the 3rd sentence say "You must create a new account or use your existing Login.gov account to sign in."?

Comment (Sgtpluck) [reply _pages/user-experience/faq-content.md:49]: seem correct! i just copy/pasted from the content doc, but does look like a word was omitted in the original

File: _pages/user-experience/faq-content.md (hunk 1 above)
Comment (Sgtpluck) [reply _pages/user-experience/faq-content.md]: fixed in https://github.com/18F/identity-dev-docs/pull/416/commits/a54ff2f891831b7429c17240343d2d83cc41b378

File: _pages/user-experience/faq-content.md (hunk 2 above)
Comment (Sgtpluck) [reply _pages/user-experience/faq-content.md]: fixed in https://github.com/18F/identity-dev-docs/pull/416/commits/a54ff2f891831b7429c17240343d2d83cc41b378

File: _pages/user-experience/faq-content.md (hunk 3 above)
Comment (Sgtpluck) [reply _pages/user-experience/faq-content.md]: fixed in https://github.com/18F/identity-dev-docs/pull/416/commits/a54ff2f891831b7429c17240343d2d83cc41b378

**Pull request merged**